### PR TITLE
tests: add tests for smb.version keyword

### PR DESCRIPTION
**Pull request by jmtaylor90**
## Ticket

If your pull request is related to a Suricata ticket, please provide
the full URL to the ticket here so this pull request can monitor
changes to the ticket status:

Redmine ticket:
https://redmine.openinfosecfoundation.org/issues/5075

### CODE REVIEW

**Comment by catenacyber** [comment]
Thanks @jmtaylor90 are you also picking up https://github.com/OISF/suricata/pull/7336 back ?

**Comment by catenacyber** [comment]
Requires https://github.com/OISF/suricata/pull/9415 apparently

**Comment by jmtaylor90** [comment]
> Requires [OISF/suricata#9415](https://github.com/OISF/suricata/pull/9415) apparently

It does, what is the way to cross reference the PRs?

**Comment by catenacyber** [comment]
> It does, what is the way to cross reference the PRs?

I copy the link of the S-V PR in the suricata one (and GitHub makes it appear in the S-V PR)

If you are interested, we use labels on S-V PRs, basically it is :
- needs suricata fix (exhibits a bug for which there is no fix)
- needs suricata PR (CI should be red now, but becomes green when PR gets merged)
- tests pass (CI should be green)

File: tests/smb-version-keyword/README.md [hunk 1]
```diff
@@ -0,0 +1,4 @@
+TEST
+====
+
+Test smb.version keyword
```
Comment (jufajardini) [inline tests/smb-version-keyword/README.md:4]: Could you please add an indication of where the pcap used here comes from? :)

Comment (jmtaylor90) [reply tests/smb-version-keyword/README.md:4]: In this case where I obtained it from the original submitters branch, it appears to be a capture from a local network they had access to. I am not sure how to denote that. How would you like that documented?

https://github.com/zer1t0/suricata-verify/tree/test/smb_version

Comment (catenacyber) [reply tests/smb-version-keyword/README.md:4]: Fair enough for me

Comment (jufajardini) [reply tests/smb-version-keyword/README.md:4]: Sorry for the late reply. A note with something along those lines - even if just - 'sample traffic from a local network' works, IMHO. :)

**Comment by jmtaylor90** [comment]
continued in #1380

**Pull request closed without merging**